WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2663,14 +2664,15 @@ public void onVideoPressInfoRequested(final String videoId) {
     }
 
     @Override
-    public String onAuthHeaderRequested(String url) {
-        String authHeader = "";
+    public Map<String, String> onAuthHeaderRequested(String url) {
+        Map<String, String> authHeaders = new HashMap<>();
+
         String token = mAccountStore.getAccessToken();
         if (mSite.isPrivate() && WPUrlUtils.safeToAddWordPressComAuthToken(url)
             && !TextUtils.isEmpty(token)) {
-            authHeader = "Bearer " + token;
+            authHeaders.put(AuthenticationUtils.AUTHORIZATION_HEADER_NAME, "Bearer " + token);
         }
```

Comment:
In `AuthenticationUtils` when `safeToAddWordPressComAuthToken` is `false` we try to use HTTP auth credentials, and it's not clear to me why we don't do that here. Similarly, we're checking `mSite.isPrivate()` here, but we don't check that in the `AuthenticationUtils`. 

I know this isn't anything you changed, I'm just curious if you know why there's the difference. If you do know why there is the difference, it might even be useful to move this logic into a method in `AuthenticationUtils` with some explanation about why the two methods are different. This is not anything that needs to block this PR.

Reply:
I'm not 100% sure about the difference there, but as far as I'm aware, `AuthenticationUtils` is mostly used with image loading, and requests that actually get to it are subject to `isPrivate()` check since it's related to Photon/https (I have only surface level of knowledge there).

I'm not sure why we are not using HTTP auth credentials in the block editor, but since it's not broken I would rather not touch it :)

Reply:
> I'm not sure why we are not using HTTP auth credentials in the block editor, but since it's not broken I would rather not touch it :)

I think that makes sense. Obviously it would be nice to figure out the cause of the inconsistency, but I don't think it's something we have to do now.